class1.md

```diff
@@ -0,0 +1,261 @@
+
+<p align="center">
+  <img align="middle" width="500" src="images/aiMlDl.png">
+</p>
+
+* AI is a term that embodies anything that enables computers to act more like humans.
+  * This is a whole field of knowledge like ‘biology’ or ‘chemistry’.
+* Machine learning is a field of study within AI.
+  * The focus of this field is on the extraction of patterns from large datasets.
+* Deep learning is an example of a collection of similar machine learning methods.
+  * It is a group of similar techniques that use neural networks to learn.
+  * There are other groups of related methods such as:
+    * support vector machines
+    * decision trees
+
+### Machine learning defined
+
+Let's look at how machine learning is defined.
+
+**[Wikipedia:](https://en.wikipedia.org/wiki/Machine_learning)** Machine learning (ML) is the study of computer algorithms that improve automatically through experience. It is seen as a subset of artificial intelligence. Machine learning algorithms build a mathematical model based on sample data, known as "training data", in order to make predictions or decisions without being explicitly programmed to do so.
+
+**[Merriam Webster:](https://www.merriam-webster.com/dictionary/machine%20learning)** the process by which a computer is able to improve its own performance (as in analyzing image files) by continuously incorporating new data into an existing statistical model.
+
+**[Nature Genetics Review:](https://www.nature.com/articles/nrg3920)** The field of machine learning includes the development and application of computer algorithms that improve with experience.
+
+While these definitions vary slightly there are a few common themes to pull out:
+
+* Machine learning is a kind of algorithm.
+  * Algorithm - A finite sequence of well-defined, computer-implementable instructions, typically to solve a class of problems or to perform a computation 
+   * Essentially it is a protocol for a computer
+* Machine learning incorporates data into a statistical model.
+* Machine learning improves with experience without human intervention.
+
+### Machine learning as code
+
+When we think of machine learning and how it applies to code we can consider machine learning to be a specific way of programming computers.
+
+* Most programming isn’t built to learn - it’s procedural
+    * This is where a programmer builds out a set of rules. This set of rules is called an algorithm. This ruleset remains static unless the coder intervenes and specifies more rules. 
+* In machine learning the algorithm is also designed and coded by a human. However, this kind of algorithm is able to learn from the data about specific parameters that will shape a statistical model for making predictions and inferences. The coder doesn’t necessarily know or control these parameters - the machine does.
+
+> **Machine learning is an algorithm (or set of rules) that self adjusts based on the large datasets that it is given**
+
+#### Challenge!
```

Comment:
Include some context here: how does this challenge relate to the rest of the material?

I'd also format the header as Challenge-pbj so you can get a unique link to it (to share with learners) when you have more than one exercise present